dasharo-compatibility/dts-update.robot

```diff
@@ -0,0 +1,94 @@
+*** Settings ***
+Library             Collections
+Library             OperatingSystem
+Library             Process
+Library             String
+Library             Telnet    timeout=20 seconds    connection_timeout=120 seconds
+Library             SSHLibrary    timeout=90 seconds
+Library             RequestsLibrary
+Resource            ../keywords.robot
+Resource            ../keys.robot
+Resource            ../rtectrl-rest-api/rtectrl.robot
+Resource            ../sonoff-rest-api/sonoff-api.robot
+Resource            ../variables.robot
+
+Suite Setup         Run Keywords
+...                     Prepare Test Suite
+Suite Teardown      Run Keyword
+...                     Log Out And Close Connection
+# This must be in Test Setup, not Suite Setup, because of a known problem
+# with QEMU: https://github.com/Dasharo/open-source-firmware-validation/issues/132
+Test Setup          Run Keywords    Make Sure That Network Boot Is Enabled    AND    Restore Initial DUT Connection Method
+
+
+*** Test Cases ***
+DTS-2 NovaCustom Dasharo v1.7.2
+    [Documentation]    This test aims to verify that Dasharo v1.7.2 on NV4x_PZ
+    ...    eligible for updates to heads with heads DES and regular update
+    ...    works as expected
+    Skip If    not ${TESTS_IN_FIRMWARE_SUPPORT}    DTS005.001 not supported
+    Skip If    not ${DTS_SUPPORT}    DTS005.001 not supported
+
+    Power On
+    Boot Dasharo Tools Suite    iPXE
+    Write Into Terminal    9
+    Read From Terminal Until Regexp    bash-\\d\\.\\d#
+
+    # We have to do this so that dts-environment doesnt ruin our mock exports
+    # I am putting this in a script to break up the command and escape
+    # robotframeworks variable expansion. Unfortunately running this as one
+    # command is impossible.
+    # This could be removed once the new dts release contains this change
+    # (PR for this is submitted already)
```

Comment:
But on another note, do we really must have these changes in release image prior running tests? We need to be able to test the changes we have locally, using QEMU as an environment. Cannot we deploy our local copy of the scripts (via SSH for instance), and then run the tests on modified scripts?